### PR TITLE
resubmission edits

### PR DESCRIPTION
**Pull request by gottacatchenall**
resolving #143

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -482,23 +479,13 @@ this extensive list of possible metrics is through the use of ROC
...
-curves. These curves are generated by considering a continuum of thresholds
+curves (see @fig:example). These curves are generated by considering a continuum of thresholds
 of classifier acceptance, and computing the values of ROC/PR metrics for
 each value of the threshold. The area-under-the-curve (AUC) is then used as
 a validation metric and are typically called AUC-ROC (Area-Under-the-Curve
 Receiver-Operator-Curve) and AUC-PR (Area-Under-the-Curve Precision-Recall)
 (_e.g._ ROC-AUC in @tbl:validation).
 
-Note that for both plots in @fig:auc, the area-under-the-curve (AUC) of the
-random model is $0.5$, and the AUC of the perfect classifier is $1.0$. This
```
Comment (TanyaS08) [inline manuscript.md:493]: I think we should keep the 0.5 is random and 1 is perfect sentiment. Just for the wise and the unwise

File: metadata.json [hunk 2]
```diff
@@ -139,7 +139,7 @@
...
...ampling these interactions is difficult. Interactions intrinsically vary across space and time, and given the number of species that compose ecological communities, it can be tough to distinguish between a true negative (where two species never interact) from a false negative (where two species have not been observed interacting even though they actually do). Assessing how likely interactions between species is a necessary imperative for several fields of ecology. This means that to predict interactions between species---and to describe the structure, variation, and change of the ecological networks they form---we need to rely on modeling tools. Here we provide a proof-of-concept, where we show a simple neural-network model makes accurate predictions about species interactions given limited data. We then  assess the challenges and opportunities associated with improving interaction predictions, and provide a conceptual roadmap forward toward predictive models of ecological networks that is explicitly spatial and temporal. We conclude with a brief primer on the relevant methods and tools needed to start building these models, which we hope will guide this research program forward.",
```
Comment (TanyaS08) [inline metadata.json:142]: Assessing how likely interactions between species is a necessary imperative for several fields -> Assessing **the likelihood of** interactions between species is a necessary ~~imperative~~ for several fields

**Review by TanyaS08 — CHANGES_REQUESTED** [review]
few comments feel free to rebuttal though

**Review by TanyaS08 — APPROVED** [review]
happy with figure changes

File: manuscript.md (hunk 1 above)
Comment (gottacatchenall) [reply manuscript.md:493]: this could go in the table, currently just says success -> 1 at the moment but could indicate 0.5 is the min

**Comment by gottacatchenall** [comment]
will merge after the build completes, then i think we can give it another read through but otherwise we're good?

File: manuscript.md (hunk 1 above)
Comment (TanyaS08) [reply manuscript.md:493]: true it is in the table so we can then just keep as is (i.e. all removed)

**Comment by TanyaS08** [comment]
works for me

**Pull request merged**